sled-agent/src/rack_setup/plan/service.rs

```diff
@@ -62,27 +62,7 @@ use uuid::Uuid;
 
 // TODO(https://github.com/oxidecomputer/omicron/issues/732): Remove
 // when Nexus provisions Clickhouse.
-// TODO(https://github.com/oxidecomputer/omicron/issues/4000): Use
-// omicron_common::policy::CLICKHOUSE_SERVER_REDUNDANCY once we enable
-// replicated ClickHouse.
-// Set to 0 when testing replicated ClickHouse.
```

Comment:
Currently we don't have the ability to deploy both single-node and replicated cluster at the same time. Do we not need to set this to 0 still if we want to deploy a replicated cluster?

Or if we deploy the replicated cluster, the single node will just run but Oximeter just won't connect to it?

Reply:
We don't actually have to do this. There is a [policy knob](https://github.com/oxidecomputer/omicron/blob/45813be40b62167eff75333c410515e8bee24211/nexus/types/src/deployment/planning_input.rs#L739-L751) that we will leave turned on for the foreseeable future that will deploy both single-node and replicated clusters. 

By the time we get rid of the single node clusters altogether, we'll just remove the initialization from RSS, turn this knob off and write a bit of extra code to ensure the single node gets expunged.

Single node management is going to be added separately to reconfigurator so we'll use that. Should be fairly straightforward.

Reply:
Oh sorry, I didn't respond to your second question. Oximeter will connect to both via DNS. @bnaecker added support for different SRV records. But that's separate from deployment concerns.

Reply:
> Single node management is going to be added separately to reconfigurator so we'll use that. Should be fairly straightforward.

I'm starting work on this, and agree that it should be straightforward.